boards/chronos/include/periph_conf.h

```diff
@@ -26,9 +26,16 @@ extern "C" {
 #endif
 
 /**
- * @brief   Clock configuration
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```

Comment:
That's what I tested too, but failed to comment due to the traps and failings of the GitHub review system ^^. I would prefer it this way. It seems like groups can't have `@todo` blocks without having this weird warnings. :-/